src/AppInstallerCLICore/ContextOrchestrator.cpp

```diff
@@ -253,12 +256,11 @@ namespace AppInstaller::CLI::Execution
 
                 std::unique_ptr<AppInstaller::ThreadLocalStorage::PreviousThreadGlobals> setThreadGlobalsToPreviousState = item->GetContext().SetForCurrentThread();
 
-                item->GetContext().GetThreadGlobals().GetTelemetryLogger().LogCommand(command->FullName());
                 command->ValidateArguments(item->GetContext().Args);
 
                 item->GetContext().EnableCtrlHandler();
 
-                terminationHR = ::AppInstaller::CLI::Execute(item->GetContext(), command);
+                ::AppInstaller::CLI::ExecuteWithoutLoggingSuccess(item->GetContext(), command.get());
             }
             WINGET_CATCH_STORE(terminationHR, APPINSTALLER_CLI_ERROR_COMMAND_FAILED);

```

Comment:
It took me a while to understand why you were removing the assignment to `terminationHR` a couple of lines above, why that doesn't affect the `if (FAILED(terminationHR))` below, and why the next conditional uses `item->GetContext().GetTerminationHR()` instead of `terminationHR`.
Can we add a comment, or rename `terminationHR` to something like `caughtHResult` so that it's clearer in the future?